php/class-ee-docker.php

```diff
@@ -78,25 +73,19 @@ public static function stop_container( $container ) {
 	 * @return bool success.
 	 */
 	public static function restart_container( $container ) {
-		return default_launch( "docker restart $container" );
+		return EE::exec( "docker restart $container" );
 	}
 
 	/**
 	 * Function to create and start the container if it does not exist.
 	 *
-	 * @param String $container Container to be created.
 	 * @param String $command   Command to launch the container.
 	 *
 	 * @return bool success.
 	 */
-	public static function create_container( $container, $command ) {
+	public static function create_container( $command ) {
 
-		$launch = EE::launch( $command, false, true );
-		default_debug( $launch );
-		if ( ! $launch->return_code ) {
-			return true;
-		}
-		EE::error( $launch->stderr );
+		return EE::exec( $command, true, true, true );
```

Comment:
what's the purpose of this? 🤔 Looks like the another wrapper of `EE::exec()` 
I don't see anything related to the container in this function.